drafts/zip-daira-disable-sprout-outputs.rst

```diff
@@ -0,0 +1,154 @@
+Sapling v4 transaction format [#v4-tx]_:
+
+- In JoinSplit descriptions, the ``ephemeralKey``, and ``encCiphertexts`` fields are
+  removed.
+
+- When creating JoinSplit descriptions, the Sprout output notes MUST be either
+  internal change (i.e. spent in a subsequent JoinSplit of the same transaction),
+  or dummy notes as described in [#protocol] section 4.7.1.
+
+- The output commitments of a JoinSplit description in a v5 transaction MUST NOT
+  be added to the output Sprout commitment tree. That is, the root of the Sprout note
+  commitment tree in the output treestate of a v5 transaction is the same as that in
+  its input treestate. This does not affect the use of interstitial Sprout treestates
+  in v5 transactions.
+
+The removal of ``ephemeralKey`` and ``encCiphertexts`` fields means that for JoinSplit
+descriptions in v5 transactions, it is not necessary for a wallet to perform trial
+decryption of those fields to detect incoming payments as described in [#protocol]_
+sections 4.18 and 4.16.2. The processing of nullifiers in the algorithm of section 4.18
+is unaffected.
+
+Note that the consensus protcol does not prevent a wallet from incorrectly creating
+a Sprout output note that is neither internal change nor a dummy note. In this case
+the value associated with the note cannot be spent and is therefore "burnt".
+
+When this proposal is activated, nodes and wallets MUST disable any facilities to
+send to Sprout addresses, and this SHOULD be made clear in user interfaces.
+
+
+Rationale
+=========
+
+This design does not require any change to the JoinSplit circuit, thus minimizing
+the risk of security regressions, and avoiding the need for a new ceremony to generate
+circuit parameters.
+
+The code changes needed are relatively small and simple, and their security is easy
+to analyse.
+
+Removing the ``ephemeralKey`` and ``encCiphertexts`` fields reduces the size of a
+JoinSplit description in a v5 transaction from 1698 bytes to 464 bytes, and avoids
+the need to scan those fields for incoming payments.
+
+During the development of this proposal, an alternative design was considered that
+would also have removed the ``commitment'' field from v5 JoinSplit descriptions,
+instead computing the commitments in a way that ensured at the consensus level that
+they had zero value. This alternative was not adopted because it would have prevented
+the use of internal change between JoinSplits, and that would have resulted in an
+unavoidable information leak of the total value of the two Sprout inputs to each
+JoinSplit. As it is, there is an unavoidable leak of the total value of Sprout inputs
+to each v5 *transaction*, but not for individual JoinSplits.
+
+
+Security and Privacy Considerations
+===================================
+
+The security motivations for making this change are described in the Motivation section.
+
+Since all clients change their behaviour at the same time when this proposal activates,
```

Comment:
nit: "after this proposal's activation height on the Zcash blockchain". We could re-org back over the activation height and so it being a step function in height not a step function in time makes a difference.